docs/docs/1.4.1/deployments/how-multi-zone-works.md

```diff
@@ -0,0 +1,31 @@
+# About multi-zone deployments
+
+Kuma supports the use case where multiple service meshes run in different zones. This use case includes hybrid deployments where you run your services on Kubernetes and in Universal mode, on VMs or elsewhere. Your mesh environment can include multiple isolated service meshes (multi-tenancy), and workloads running in different regions, on different clouds, or in different datacenters. A zone can be a Kubernetes cluster, a VPC, or any other deployment you need to include in the same distributed mesh environment.
```

Comment:
> Kuma supports running your service mesh in multiple zones. It is even possible to run with a mix of Kubernetes and Universal zones.
Probably shorter and simpler.

Reply:
While I agree it's shorter and simpler and the one above is wordy, those 2 sentences do not explain what the zone is.

Reply:
Yeah, some of this belongs in a higher-level "about Kuma" topic, not here. Originally I was trying to trim and associate a bunch o' different bits of explanation from the current live doc. How about something like

```suggestion
Kuma supports the a service mesh that is distributed across multiple zones. Different zones in the same mesh can run on Kubernetes or in Universal mode, in different regions, on different cloud providers, or in different datacenters. The only thing that zones have in common is their connection to the same global control plane.
```